lib/src/presentation/widget/pokemon_data.dart

```diff
@@ -0,0 +1,165 @@
+              child: Container(
+                height: Constants.detailsNameHeight,
+                width: Constants.detailsNameWeight,
+                child: FittedBox(
+                  child: Text(
+                    pokemonEntity.name.toUpperCase(),
+                    style: TextStyles.detailsNameTextStyle,
+                  ),
+                ),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(Constants.detailsPadding),
+              child: Container(
+                decoration: BoxDecoration(
+                  border: Border.all(
+                    color: ColorsConstants.detailsCaptureRateBorderColor,
+                    width: Constants.detailsBoxBorderWidth,
+                  ),
+                  borderRadius: BorderRadius.circular(
+                    Constants.detailsBoxBorderRadius,
+                  ),
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.all(Constants.detailsPadding),
+                      child: Text(
+                        '${Constants.detailsWeightLabel} ${pokemonEntity.weight}',
+                        style: TextStyles.detailsWeightAndHeightTextStyle,
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(Constants.detailsPadding),
+                      child: Text(
+                        '${Constants.detailsHeightLabel} ${pokemonEntity.height}',
+                        style: TextStyles.detailsWeightAndHeightTextStyle,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(Constants.detailsPadding),
+              child: Container(
+                decoration: BoxDecoration(
+                  color: ColorsConstants.detailsAttributesBoxColor,
+                  border: Border.all(
+                    color: Colors.black,
+                    width: Constants.detailsBoxBorderWidth,
+                  ),
+                  borderRadius: BorderRadius.circular(
+                    Constants.detailsBoxBorderRadius,
+                  ),
+                ),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.symmetric(
+                          horizontal: Constants.detailsBoxPadding),
+                      child: Text(
+                        '${Constants.detailsBabyLabel} ${pokemonEntity.species.isBaby ? Constants.yesValue : Constants.noValue}',
+                        style: TextStyles.detailsAttributesTextStyle,
+                      ),
+                    ),
```

Comment:
this is used multiple times, you can create a widget to avoid boilerplate

Reply:
solved